<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leditPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ZipWidget" name="browserWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>操作(&amp;O)</string>
    </property>
    <addaction name="actionUnCompressAs"/>
    <addaction name="actionAdd"/>
    <addaction name="actionInfo"/>
    <addaction name="actionComments"/>
    <addaction name="actionTestZipFile"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionManagePwd"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperate"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save&amp;As</string>
   </property>
   <property name="iconText">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionUnCompressAs">
   <property name="text">
    <string>UnCompress&amp;As</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add To Zip File</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Infomation</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionComments">
   <property name="text">
    <string>Co&amp;mments</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionTestZipFile">
   <property name="text">
    <string>&amp;Test Zip File</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actionManagePwd">
   <property name="text">
    <string>Manage&amp;Pwd</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZipWidget</class>
   <extends>QWidget</extends>
   <header>zipwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
